intake/source/cache.py

```diff
@@ -31,6 +31,9 @@ def sanitize_path(path):
     return path
 
 
+display = {}
```

Comment:
Why is ``display`` a dict key with ints instead of an array?

Reply:
Should probably be a set? I was using it differently before, but the values are no longer useful.

Reply:
Yeah, based on the context it which it is used, I think a set would work better.